src/runtime_src/core/edge/drm/zocl/zocl_aie.c

```diff
@@ -0,0 +1,102 @@
+/* SPDX-License-Identifier: GPL-2.0 OR Apache-2.0 */
+/*
+ * A GEM style (optionally CMA backed) device manager for ZynQ based
+ * OpenCL accelerators.
+ *
+ * Copyright (C) 2020 Xilinx, Inc. All rights reserved.
+ *
+ * Author(s): Himanshu Choudhary <hchoudha@xilinx.com>
+ *
+ * This file is dual-licensed; you may select either the GNU General Public
+ * License version 2 or Apache License, Version 2.0.
+ */
+
+#include "ert.h"
+#include "zocl_drv.h"
+#include "zocl_aie.h"
+#include "sched_exec.h"
+
+int
+zocl_aie_getcmd_ioctl(struct drm_device *dev, void *data, struct drm_file *filp)
+{
+	struct drm_zocl_dev *zdev = dev->dev_private;
+	struct aie_info *aie = zdev->aie_information;
+	struct aie_info_cmd *acmd;
+	struct drm_zocl_aie_cmd *kdata = data;
+
+	// If no command, the process who calls this ioctl will block here
```

Comment:
Please use c89 /* */ other than c99 comment style in kernel code

Reply:
done